src/hokusai/hokusai.yml

```diff
@@ -1,12 +1,12 @@
 # Orb Version 0.1.0
 
-version: 2.1
+version: 2.2
 description: Reusable hokusai tasks for managing deployments
 
 executors:
   deploy:
     docker:
-      - image: artsy/hokusai:0.5
+      - image: artsy/hokusai:0.5.6
```

Comment:
Is there any harm leaving this number fuzzy as `0.5`?

Reply:
It sounds like we update the orb version on a repo-by-repo basis so I'd assume this is still safe as long as bumping orbs in individual projects is not paired with other changes.